examples/rustls-mtls/Cargo.toml

```diff
@@ -7,10 +7,9 @@ authors = ["Rick Richardson <rick.richardson@gmail.com>", "AWS s2n"]
 [dependencies]
 # Remove the `provider-tls-default` feature and add `provider-tls-rustls` in order to use the rustls backend
 s2n-quic = { version = "1", path = "../../quic/s2n-quic", default-features = false, features = ["provider-address-token-default", "provider-tls-rustls"] }
+rustls = "0.20"
```

Comment:
We don't need rustls here either. It's also exported in the provider module

Reply:
Ahh, I had tried to `use s2n_quic::provider::tls::rustls` but missed the wildcard import. Looks like `use s2n_quic::provider::tls::rustls::rustls` is the correct import.